src/types/queryTypes.ts

```diff
@@ -0,0 +1,14 @@
+import { Facet } from '../results/ResultsContainer';
+
+type Response = {
```

Comment:
Just wondering if `queryTypes` is the right filename here as it is the response?

Reply:
renamed to `responseTypes`